cli/cmd/sync_v0_1.go

```diff
@@ -1,162 +0,0 @@
-package cmd
```

Comment:
I deleted this old version and just returned an error for users to upgrade source or use an older CLI.